packages/vite/src/node/server/pluginContainer.ts

```diff
@@ -201,7 +201,8 @@ export async function createPluginContainer(
     for (const plugin of getSortedPlugins(hookName)) {
       const hook = plugin[hookName]
       if (!hook) continue
-      // @ts-expect-error hook is not a primitive
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore hook is not a primitive
```

Comment:
Why not using `@ts-expect-error` 🤔?

Reply:
It's because some code will just get generated when the project is build and such things
Vite's code base uses `@ts-ignore` a lot because of this